tests/fixtures/inbox_data.py

```diff
@@ -96,6 +97,18 @@ def inbox_data(mlwhdb_test_session):
 
     mlwhdb_test_session.commit()
 
+    yield True
+
+    print("\nTEARDOWN inbox_data")
+    mlwhdb_test_session.execute(text("SET foreign_key_checks=0;"))
```

Comment:
Is it right that `mlwhdb_test_session` is created for each test function? There is no `scope="module"` for this fixture (line 98 in conftest.py)

Reply:
Thanks for picking this up. Fixed now.